broker/src/test/java/io/zeebe/broker/system/configuration/BrokerCfgTest.java

```diff
@@ -892,4 +892,15 @@ private void assertSystemClusterConfiguration(
     assertThat(cfgCluster.getClusterSize()).isEqualTo(clusterSize);
     assertThat(cfgCluster.getInitialContactPoints()).isEqualTo(initialContactPoints);
   }
+
+  private void assertWithDefaultConfigurations(final Consumer<BrokerCfg> assertions) {
+    Stream.of("default", "empty")
+        .forEach(configFileName -> assertWithDefaultConfiguration(assertions, configFileName));
+  }
+
+  private void assertWithDefaultConfiguration(
```

Comment:
This could be renamed:
```suggestion
  private void assertWithConfiguration(
```